bgs/bgs.go

```diff
@@ -661,3 +663,24 @@ func (s *BGS) createExternalUser(ctx context.Context, did string) (*models.Actor
 
 	return subj, nil
 }
+
+func (bgs *BGS) TakedownRepo(ctx context.Context, did string) error {
```

Comment:
"take down" (with a space) is a phrasal verb. "takedown" is the noun. 

So should this be "TakeDownRepo" instead? (and elsewhere)

Reply:
you have reached the limits of my mastery of english. I *think* weve been calling them 'takedowns' elsewhere? cc @bnewbold

Reply:
You can call it "takedown" when it's a noun... like `(*BGS).GetRepoTakedowns()` returning nouns (hypothetical `Takedown` objects). But when you're telling something to do something (an imperative verb: "take down") then it'd be `(*BGS).TakeDownRepo`

"Clean up this room!" "Nice cleanup." etc

But maybe it's a losing battle when you're also working with lots of generated code anyway that's not going to be grammatical.

Reply:
I agree with bradfitz that `TakeDownRepo()` is more correct. In the noun form would have to add a verb like `CreateRepoTakedown()`.